### ISSUE TITLE
apply fix does not work for sort properties

### ISSUE THREAD

**Issue by calderJamNet** [brigade/scss-lint#822]
I get the following error:

Browser Error

The URL file://bred.txt#invokelocal:0 does not refer to a file.

I'm using scss-lint with the intellij plugin.

**Comment by sds** [comment]
Hey @calderJamNet,

The information you've provided makes it very difficult to determine if this is indeed a problem with SCSS-lint or some other issue. It would be immensely helpful if you could provide a snippet of code that reproduces the issue.

For now, I'm closing this. Feel free to reopen if you can provide a working example. Thanks!

**Comment by calderJamNet** [comment]
@sds sorry for the lack of info, I didn't include any code because I figured it would be irrelevant as the problem effects any and all css/scss code.

I also just realized that the "sort properties" fix does work if you do it directly by hovering over the highlighted code and clicking intellij's lightbulb popup thing, but it still does not work from the from the inspect window after "inspect code" has been run. It would be great if I could get this to work as It would allow me to remedy many warnings at the same time.

Heres a code snippet:

```
  #visibility-dropdown {
    margin: 0;
    padding: 0;
    width: 7rem;
    &-input {
      height: 2.46rem;
      min-height: 0;
      paSCdding: .6rem;
      resize: none;
      vertical-align: bottom;
    }
}
```

heres a video to better show the what I'm talking about.

https://vid.me/zWAG

_I forgot to include the error popup I originally described. This happens when you click on "sort properties" under "Problem resolution" on the right hand column of the inspect window._

Also is there a list of "fixes" that should work? Because many of them don't seem to do anything, while others are not offered. For example:

* `shorthand property for margin should be written more concisely...` does not offer a fix with the lightbulb dropdown, but in the inspection window it shows the option to "sort properties", which does nothing.

* `rule declaration should be preceded by an empty line...` does not even highlight the code as containing an warning, however will show up in the inspection window if a code inspection is run. Like the example above, this offers the fix "sort properties" which doesn't do anything. Not sure if the fix is just called "sort properties" and should work but isn't because of the original issue, or if there is no fix/the wrong fix is being offered.

**Comment by sds** [comment]
Thanks for providing some additional information, @calderJamNet.

The SCSS-Lint project does not support third-party plugins to editors. I'm not sure which plugin implements the "sort properties" feature you demonstrate in the video, but it's definitely _not_ an SCSS-Lint feature.

Furthermore, when I run the latest version of `scss-lint` (version 0.49.0 at time of writing) against the code sample you provide:

```scss
#visibility-dropdown {
  margin: 0;
  padding: 0;
  width: 7rem;
  &-input {
    height: 2.46rem;
    min-height: 0;  
    padding: .6rem;
    resize: none;
    vertical-align: bottom;
  }
}
```

...I get valid lints reported (and no errors):

```
test.scss:1:1 [W] IdSelector: Avoid using id selectors
test.scss:5:1 [W] EmptyLineBetweenBlocks: Rule declaration should be preceded by an empty line
```

Again, if you can provide a simple file that when running `scss-lint my-file.scss` causes a crash, I'm happy to look into it. Thanks!
